README.md

```diff
@@ -1,8 +1,150 @@
+<div align="center">
+  <a href="https://novu.co" target="_blank">
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/2233092/213641039-220ac15f-f367-4d13-9eaf-56e79433b8c1.png">
+    <img src="https://user-images.githubusercontent.com/2233092/213641043-3bbb3f21-3c53-4e67-afe5-755aeb222159.png" width="280" alt="Logo"/>
+  </picture>
+  </a>
+</div>
+
 # Novu Java SDK
 
+[![License](https://poser.pugx.org/unicodeveloper/novu/license.svg)](LICENSE.md)
+
 Novu's API exposes the entire Novu features via a standardized programmatic interface. Please refer to the full [documentation](https://docs.novu.co/docs/overview/introduction) to learn more.
 
+## Installation
+
+```xml
+// add dependency
+<dependency>
+    <groupId>co.novu</groupId>
+    <artifactId>novu-java</artifactId>
+    <version>1.0.0</version>
+</dependency>
```

Comment:
Please add that of Gradle. It'd be:
**implementation 'co.novu:novu-java:1.0.0'**